### ISSUE TITLE
Add `vCHORD` to oncoanalyser for HRD detection in targeted mode

### ISSUE THREAD

**Issue by bounlu** [nf-core/oncoanalyser#118]
### Description of feature

In the `targeted` mode, CHORD is disabled and [vCHORD](https://github.com/hartwigmedical/hmftools/tree/master/v-chord) is used instead for HRD detection for a panel of samples. However, it's not implemented in nf-core/oncoanalyser yet. It will be great to have it here too. Thanks.

**Comment by scwatts** [comment]
Agreed, this will be implemented in oncoanalyser in due course! I'll leave the issue open until that piece of work is complete
